spec/views/calculators/calculator.html.erb_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Calculator, type: :view do
+  describe "rendering text directly" do
+    it "displays the given text" do  
+      render :calculator => "This is directly rendered"  
+      rendered.should contain("directly rendered")
+    end
+  end
+end
```

Comment:
you can skip this test. we will cover this funcitonality later

Reply:
Thanks, done.